### PR TITLE
feat(Forms): add validation to Field.BankAccountNumber

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-eufemia/src/extensions/forms/Field/BankAccountNumber/BankAccountNumber.tsx [hunk 1]
```diff
@@ -1,22 +1,60 @@
...
 
-export type Props = StringFieldProps & {
+export type Props = Omit<StringFieldProps, 'onBlurValidator'> & {
   validate?: boolean
   omitMask?: boolean
+  onBlurValidator?: Validator<string> | false
 }
 
 function BankAccountNumber(props: Props) {
   const translations = useTranslation().BankAccountNumber
```
Comment (joakbjerk) [inline packages/dnb-eufemia/src/extensions/forms/Field/BankAccountNumber/BankAccountNumber.tsx:15]:
```suggestion
  const {
    errorBankAccountNumber,
    errorBankAccountNumberLength,
    errorRequired,
    label,
  } = useTranslation().BankAccountNumber
```

Just a nitpicky thought, that should not change functionality at all 😄 but could we remove the restructuring on line 17 to 22 and move it up to the useTranslation hook line? 🤔

**Review by joakbjerk — APPROVED** [review]
Looks good 🔥 Works as intended when testing in the preview portal 👍

File: packages/dnb-eufemia/src/extensions/forms/Field/BankAccountNumber/BankAccountNumber.tsx (hunk 1 above)
Comment (langz) [reply packages/dnb-eufemia/src/extensions/forms/Field/BankAccountNumber/BankAccountNumber.tsx]: Done done.

**Review by tujoworker — APPROVED** [review]
Looks good 👏

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/BankAccountNumber/demos.mdx [hunk 2]
```diff
@@ -41,3 +41,9 @@ import * as Examples from './Examples'
 ### Validation - Required
 
 <Examples.ValidationRequired />
+
+### Extend validation with custom validation function
+
+You can [extend the existing validation](/uilib/extensions/forms/create-component/useFieldProps/info/#validators)(`bankAccountNumberValidator`) with your own validation function.
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/BankAccountNumber/demos.mdx:47]:
```suggestion
You can [extend the existing validation](/uilib/extensions/forms/create-component/useFieldProps/info/#validators) (`bankAccountNumberValidator`) with your own validation function.
```

😉

**Pull request merged**